website/src/components/HomepageFeatures/index.tsx

```diff
@@ -108,14 +110,24 @@ function CodePreview(props: { title: string; code: string }) {
 
 function Code(props: { code: string }) {
   return (
-    <SyntaxHighlighter
+    <Highlight
+      {...highlightDefaultProps}
+      theme={theme}
+      code={props.code}
       language="typescript"
-      style={a11yDark}
-      wrapLongLines={false}
-      customStyle={{ width: "100%" }}
     >
-      {props.code}
-    </SyntaxHighlighter>
+      {({ className, style, tokens, getLineProps, getTokenProps }) => (
+        <pre className={className} style={style}>
+          {tokens.map((line, i) => (
+            <div {...getLineProps({ line, key: i })}>
+              {line.map((token, key) => (
+                <span {...getTokenProps({ token, key })} />
+              ))}
+            </div>
+          ))}
+        </pre>
+      )}
+    </Highlight>
```

Comment:
There is a functionless highlighter component that includes all of this and correctly highlights Function

Reply:
https://github.com/functionless/functionless/blob/main/website/src/components/highlighter.tsx#L22